Client/Frontend/Autofill/CreditCard/ViewComponents/ToastView.swift

```diff
@@ -0,0 +1,77 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+import SwiftUI
+
+struct ToastView: View {
+    enum MessageType {
+        case savedCard
+        case updatedCard
+        case removedCard
+
+        var message: String {
+            switch self {
+            case .savedCard: return String.CreditCard.SnackBar.SavedCardLabel
+            case .updatedCard: return String.CreditCard.SnackBar.UpdatedCardLabel
+            case .removedCard: return String.CreditCard.SnackBar.RemovedCardLabel
+            }
+        }
+    }
+
+    var textColor: Color = .white
+    var backgroundColor: Color = .blue
+    var messageType: MessageType
+    @State var isShowing = false
+
+    var toast: some View {
+        VStack {
+            Spacer()
+            Text(messageType.message)
+                .frame(minWidth: 0, maxWidth: .infinity)
+                .padding()
+                .background(backgroundColor)
+                .foregroundColor(textColor)
+        }
+    }
+
+    var body: some View {
+        VStack {
+        }
```

Comment:
Any benefit of using empty VStack? 🤔

Reply:
It's just a wrapper to use the modifier, I had to add it to use it from UIKit, to use it in SWiftUI we can call directly to the modifier